index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <header>
+        <nav>
+          <a class="logo" href="">My Logo</a>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="Menu">
+          </div>
+      </nav>
+      <section>
+        <div class="topimg">
+            <div class="imgorder imgorder1">
+              <img class="backimg backimg1" src="./images/Ellipse 11.png" alt="">
+              <img class="backimg backimg2" src="./images/Rectangle 4.png" alt="">
+            </div>
+            <div class="imgorder imgorder2">
+              <img class="backimg backimg3" src="./images/Rectangle 7.png" alt="">
+              <img class="backimg backimg4" src="./images/Ellipse 5.png" alt="">
+            </div>
+            <img class="icon" src="./images/Ellipse 7.png" alt="">
+            <div class="imgorder imgorder3">
+              <img class="backimg backimg5" src="./images/Rectangle 4.png" alt="">
+              <img class="backimg backimg6" src="./images/Ellipse 3.png" alt="">
+            </div>
+        </div>
+        <h1>I’m kirubel Glad <br/> to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>LET’S CONNECT</p>
+        <!-- <img src="./images/social media icons.png" alt=""> -->
+        <ul class="social">
+          <li><a href="#"><img src="./images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/" alt=""></a></li>
+          <li><a href="#"><img src="./images/Linkedin icon.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/github.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/angellist.png" alt=""></a></li>
```

Comment:
- Good job putting these in a list item however they are not properly styled. Kindly style theses as shown in the Figma design. 
- I noticed that you have applied these styles to your `.socials` class. 
```
display: inline-block;
    width: 100%;
```
Kindly remove these. I think they are part of the reason why you're not being able to style them properly :100: 
- I will suggest you use either `display: flex` or `grid` to style them instead of applying `display: inline-block` to each of them :+1: 

Your Design
![image](https://user-images.githubusercontent.com/102699958/219857480-598ab547-7d81-4395-a240-fa9f0cfeff59.png)

The Figma Design
![image](https://user-images.githubusercontent.com/102699958/219857515-ebb0d78c-49f4-4674-aaec-1afd18c4ae50.png)